generators/app/templates/src/app/components/RadioGroup/index.test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import RadioOption from './components/RadioOption';
+
+import RadioGroup from './index';
+
+describe('#RadioGroup', () => {
+  let selectedOption = null;
+  let component = null;
+  const onChange = event => (selectedOption = event.target.value);
+  const getInput = (comp, id) => comp.find(`[data-testid="radioButton-${id}"]`);
+
+  beforeEach(() => {
+    selectedOption = 'option2';
+  });
+
+  describe('when the group is disabled', () => {
+    beforeEach(() => {
+      component = mount(
+        <RadioGroup onChange={onChange} initialValue={selectedOption} name="radioButton" disabled>
+          <RadioOption id="option1" label="Option 1" />
+          <RadioOption id="option2" label="Option 2" />
+        </RadioGroup>
+      );
+    });
+
+    it('all the inputs are disabled', () => {
+      const selectedInput1 = getInput(component, 'option1');
+      const selectedInput2 = getInput(component, 'option2');
+      expect(selectedInput1.props().disabled).toBeTruthy();
+      expect(selectedInput2.props().disabled).toBeTruthy();
+    });
+
+    it('matches last snapshot', () => {
+      expect(renderer.create(component).toJSON()).toMatchSnapshot();
```

Comment:
May be we should consider using [enzyme-to-json](https://github.com/adriantoine/enzyme-to-json) to avoid mixing testing libraries. 
This way,
```js
expect(component).toMatchSnapshot();
```
would work properly.
It adds another dependency but makes it the tests a little easier to understand and devs don't need to be thinking which library is useful for each situation.
I tried it and every test worked correctly (snapshots have to be updated).
Tell me what you think @damfinkel @pabloferro 
This is the basic config that needs to be added
```js
// npm install --save-dev enzyme-to-json
"jest": {
  "snapshotSerializers": [
    "enzyme-to-json/serializer"
  ]
}
```

Reply:
this seems waaay cleaner. How heavy is the dependency? We'd need to refactor most if not all tests too though

Reply:
I like it, let's add a task to do so